layouts/partials/meta.html

```diff
@@ -1,10 +1,14 @@
-<title>{{if .IsHome}}{{ .Title }}{{else}}{{ .Title }} - {{ .Site.Title }}{{end}}
+<title>
+  {{ if .IsHome }}
+      {{ .Site.Title }} - Food Spraying Technology
+  {{ else }}
+      {{ .Title }} - {{ .Site.Title }}
+  {{ end }}
 </title>
-<meta name="description"
-  content="{{ if .Description }}{{ .Description }}{{ else }}{{ .Site.Params.description }}{{ end }}" />
-<meta itemprop="name" content="{{ .Title }} - {{ .Site.Title }}" />
-<meta name="application-name" content="{{ .Title }} - {{ .Site.Title }}" />
-
-{{ template "_internal/schema.html" }}
+<meta name="description" content="{{ with .Description }}{{ . }}{{ else }}{{ .Site.Params.description }}{{ end }}">
+<link rel="canonical" href="{{ .Permalink }}" />
+<meta itemprop="name" content="{{ if .IsHome }}{{ .Site.Title }} - Food Spraying Technology{{ else }}{{ .Title }} - {{ .Site.Title }}{{ end }}" />
+<meta name="application-name" content="{{ if .IsHome }}{{ .Site.Title }} - Food Spraying Technology{{ else }}{{ .Title }} - {{ .Site.Title }}{{ end }}" />
```

Comment:
Remove hard coded - Food Spraying Technology. It should match the same as the title tag